README.md

```diff
@@ -20,35 +22,20 @@ faster than other packages according to public benchmarks.
 ## Table of contents
 
 * [Installation](#installation)
-* [Hello World](#hello-world)
-* [API reference](#api-reference)
-* [Walking\-through example in Go](#walking-through-example-in-go)
-* [Help](#help)
-* [Usage](#usage)
-* [Schema](#schema)
-* [Custom (un)packing and typed selects and function calls](#custom-unpacking-and-typed-selects-and-function-calls)
-* [Options](#options)
-* [Working with queue](#working-with-queue)
-* [Tests](#tests)
+* [Documentation](#documentation)
+  * [API reference](#api-reference)
+  * [Walking\-through example](#walking-through-example)
+* [Contributing](#contributing)
 * [Alternative connectors](#alternative-connectors)
 
 ## Installation
 
-We assume that you have Tarantool version 1.6 and a modern Linux or BSD
+We assume that you have Tarantool version 1.6+ and a modern Linux or BSD
 operating system.
 
-You will need a current version of `go`, version 1.3 or later (use
-`go version` to check the version number). Do not use `gccgo-go`.
-
-**Note:** If your `go` version is younger than 1.3, or if `go` is not installed,
-download the latest tarball from [golang.org](https://golang.org/dl/) and say:
-
-```bash
-$ sudo tar -C /usr/local -xzf go1.7.5.linux-amd64.tar.gz
-$ export PATH=$PATH:/usr/local/go/bin
-$ export GOPATH="/usr/local/go/go-tarantool"
-$ sudo chmod -R a+rwx /usr/local/go </pre>
-```
+You will need a current version of `go`, version 1.13 or later (use `go version`
```

Comment:
You need a current version of 'go', version 1.13 or later (use 'go version'
to check the version number). Do not use 'gccgo-go'. Learn more about installing Go at [official site](https://go.dev/doc/install).

Reply:
Updated.